airflow/providers/google/README.md

```diff
@@ -566,6 +567,8 @@ in [Naming conventions for provider packages](https://github.com/apache/airflow/
 
 | Commit                                                                                         | Committed   | Subject                                                                                                                                                            |
 |:-----------------------------------------------------------------------------------------------|:------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [c7e5bce57](https://github.com/apache/airflow/commit/c7e5bce57fe7f51cefce4f8a41ce408ac5675d13) | 2020-06-19  | Prepare backport release candidate for 2020.6.23rc1 (#9370)                                                                                                        |
```

Comment:
Can you add this hotfix? 
https://github.com/apache/airflow/pull/9390

Reply:
It's not there. This is general setup change so it will not appear here. It's automatically derived from any change in provider's file.

Reply:
I was mainly worried about the table with dependencies, but now I see that Apache Beam is not included there, so LGTM
https://github.com/apache/airflow/blob/9ce36ae5db03782795d0b0990b09b26b29ad6f0a/airflow/providers/google/README.md#pip-requirements

Reply:
Apache-beam is excluded as prerequisite from providers-google (on your request) so providers-google have no dependency at all for apache-beam (it can be installed separately). The change only affected [apache-beam] extra.